README.md

```diff
@@ -617,6 +617,7 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [DopeUI](https://dopeui.co/): High quality website UI templates
 - [Coolors](https://coolors.co/): The super fast color schemes generator
 - [uiGradients](https://uigradients.com/): An awesome gradient generator
+- [glassmorphism generator](https://ui.glass/generator/): Amazing tool to generate glassmorphism design.
```

Comment:
Can you capitalize the first letter of each word?

Reply:
:) Thank you, yes I will do it right away